launcher/FileSystem.cpp

```diff
@@ -162,6 +162,35 @@ bool ensureFolderPathExists(QString foldernamepath)
     return success;
 }
 
+/// @brief Copy file at src to dest, ensures the full filepath exsists
+/// @param src srouce file path
+/// @param dst destination file path
+/// @return boolean: was there an error during the filecopy?
+bool copyFile(QString &src, QString &dst) {
```

Comment:
I've fixed the issue of `FS::copy` not copying individual files in #333, so this can go once that is merged :p

Reply:
Good to know, I've added a `FIXME\TODO` comment. Once #333 is merged I'll make the change.